### ISSUE TITLE
Implement slot-2 open bus behavior

### ISSUE THREAD

**Issue by rzumer** [Arisotura/melonDS#577]
This behavior is missing from [here](https://github.com/Arisotura/melonDS/blob/master/src/NDS.cpp#L1701) and other memory access functions. It's already marked as TODO in code but I figured an issue would help track progress and add some details (and maybe remind me to do it).

The expected values are described in GBATEK [here](https://problemkaputt.de/gbatek.htm#dsmemorycontrolcartridgesandmainram).

Note: on DSi the current behavior (zero memory) is correct, so the DSi branch should take that into account.

**Comment by Arisotura** [comment]
isn't there a story like how it's different between DSPhat and DSLite too

**Comment by rzumer** [comment]
I haven't read anything suggesting that, but who knows. I don't have an original DS myself to test unfortunately.

**Comment by MojoJojoDojo** [comment]
> isn't there a story like how it's different between DSPhat and DSLite too

Yeah you can read about it here : https://mgba.io/2017/05/29/holy-grail-bugs/#mega-man-battle-network-4
